app/javascript/tests/layout/Breadcrumb.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Breadcrumb from '../../components/layout/Breadcrumb';
+import { mockBreadcrumbs1 } from '../mocks/mockBreadcrumbs';
+import { mockJurisdiction1 } from '../mocks/mockJurisdiction';
+
+describe('Breadcrumb', () => {
+  it('Properly renders breadcrumb text and links and jurisdiction', () => {
+    const wrapper = shallow(<Breadcrumb crumbs={mockBreadcrumbs1} jurisdiction={mockJurisdiction1.path} />);
+    mockBreadcrumbs1.forEach((crumb, index) => {
+      if (crumb.href === null) {
+        expect(wrapper.find('li').at(index).text()).toEqual(crumb.value);
```

Comment:
can you just add in here a check that insures it's not a link?

Reply:
https://github.com/SaraAlert/SaraAlert/pull/937/commits/5b32a8a7009cc9386a71b5e1ff1e445041b2e34e